src/main/java/net/fabricmc/fabric/api/client/model/ModelProvider.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2016, 2017, 2018 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.model;
+
+import com.sun.istack.internal.Nullable;
```

Comment:
Shouldn't be using this `Nullable`, should be the `javax.annotation` one (which, for some reason, seems not to be present).

Reply:
This was the one that was suggested: https://github.com/apiguardian-team/apiguardian

Looks fine to me.